### PR TITLE
docs: add azure-functions CN document

### PR DESCRIPTION
**Pull request by penghuima**
Signed-off-by: penghuima <2992426202@qq.com>

### Description

docs: add azure-functions CN document

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/azure-functions.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`azure-functions` 插件用于将 [Azure Serverless Function](https://azure.microsoft.com/en-in/services/functions/) 作为动态上游集成至 APISIX，从而实现将访问指定 URI 的请求代理到 Microsoft Azure 云。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:31]:
```suggestion
`azure-functions` 插件用于将 [Azure Serverless Function](https://azure.microsoft.com/en-in/services/functions/) 作为动态上游集成至 APISIX，从而实现将访问指定 URI 的请求代理到 Microsoft Azure 云服务。
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+启用 `azure-functions` 插件后，该插件会终止对已配置 URI 的请求，并代表客户端向 Azure Functions 发起一个新的请求。这个新请求中携带了之前配置的授权详细信息，包括请求头、请求体和参数（以上参数都是从原始请求中传递的），然后 `azure-functions` 插件会将带有响应头、状态码和响应体的响应信息返回给使用 APISIX 发起请求的客户端。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:33]:
```suggestion
启用 `azure-functions` 插件后，该插件会终止对已配置 URI 的请求，并代表客户端向 Azure Functions 发起一个新的请求。该新请求中携带了之前配置的授权详细信息，包括请求头、请求体和参数（以上参数都是从原始请求中传递的）。之后便会通过 `azure-functions` 插件，将带有响应头、状态码和响应体的信息返回给使用 APISIX 发起请求的客户端。
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+## 属性
+
+| 名称                   | 类型    | 必选项 | 默认值 | 有效值     | 描述                                                         |
+| ---------------------- | ------- | ------ | ------ | ---------- | ------------------------------------------------------------ |
+| function_uri           | string  | 是     |        |            | 触发 Serverless Functions 的 Azure Functions 端点。例如，`http://test-apisix.azurewebsites.net/api/HttpTrigger`。 |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:39]:
```suggestion
| function_uri           | string  | 是     |        |            | 触发 Serverless Functions 的 Azure Functions 端点。例如`http://test-apisix.azurewebsites.net/api/HttpTrigger`。 |
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+| keepalive_timeout      | integer | 否     | 60000  | [1000,...] | 当连接空闲时，保持该连接处于活动状态的时间（以毫秒为单位）。 |
+
+## 元数据
+
+| 名称            | 类型   | 必选项 | 默认值 | 描述                                                         |
+| --------------- | ------ | ------ | ------ | ------------------------------------------------------------ |
+| master_apikey   | string | 否     | ""     | 可用于访问 Azure Functions URI 的 API 密钥。                 |
+| master_clientid | string | 否     | ""     | 可用于授权 Azure Functions URI 的客户 ID（Active Directory）。 |
+
+`azure-functions` 插件的元数据提供了授权回退的功能。它定义了 `master_apikey` 和 `master_clientid` 字段，用户可以为关键任务的应用部署声明主 API 密钥或客户端 ID。因此，如果在 `azure-functions` 插件属性中没有找到授权凭证，元数据中的授权凭证就会发挥作用。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:56]:
```suggestion
`azure-functions` 插件的元数据提供了授权回退的功能。它定义了 `master_apikey` 和 `master_clientid` 字段，用户可以为关键任务的应用部署声明主 API 密钥或客户端 ID。因此，如果在 `azure-functions` 插件属性中没有找到相关授权凭证，此时元数据中的授权凭证就会发挥作用。
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+授权方式优先级排序如下：
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:58]:
```suggestion
:::note

授权方式优先级排序如下：
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+- 首先，`azure-functions` 插件在 Apache APISIX 代理的请求头中寻找 `x-functions-key` 或 `x-functions-clientid` 键。
+- 如果没有找到，`azure-functions` 插件会检查插件属性中的授权凭证。如果授权凭证存在，`azure-functions` 插件会将相应的授权标头添加到发送到 Azure cloud function 的请求中。
+- 如果在 `azure-functions` 插件属性中没有找到授权凭证，Apache APISIX 将获取插件元数据配置并使用主密钥。
+
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:63]:
```suggestion
:::

```

File: docs/zh/latest/plugins/azure-functions.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+如果你想添加一个新的主 API Key，请向 `/apisix/admin/plugin_metadata` 端点发出请求，并附上所需的元数据。示例如下：
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:64]: 这里的 API Key，包括前文的 `主 API 密钥`、`主密钥` 等用词，是否需要统一一下。如果是同一个意思，建议统一保留一种格式

File: docs/zh/latest/plugins/azure-functions.md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+```shell
+curl http://127.0.0.1:9080/apisix/admin/plugin_metadata/azure-functions -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "master_apikey" : "<Your Azure master access key>"
+}'
+```
+
+## 启用插件
+
+以下示例展示了如何在指定路由上启用 `azure-functions` 插件。请确保你的 Azure Functions 已经部署好并提供服务。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:75]:
```suggestion
以下示例展示了如何在指定路由上启用 `azure-functions` 插件。请确保你的 Azure Functions 已提前部署好，并正常提供服务。
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+content-type: text/plain; charset=utf-8
+...
+Hello, APISIX
+```
+
+### 配置路径转发
+
+`azure-functions` 插件在代理请求到 Azure Functions 上游时也支持 URL 路径转发。基本请求路径的扩展被附加到插件配置中指定的 `function_uri` 字段上。
+
+:::info IMPORTANT
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/azure-functions.md:142]:
```suggestion
:::info 注意
```

Comment (SylviaBABY) [reply docs/zh/latest/plugins/azure-functions.md:142]: reference this, so here in Chinese content, we need use `注意`, and in EN version you can use `IMPORTANT`
image

File: docs/zh/latest/plugins/azure-functions.md [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+
+`azure-functions` 插件的元数据提供了授权回退的功能。它定义了 `master_apikey` 和 `master_clientid` 字段，用户可以为关键任务的应用部署声明 API 密钥或客户端 ID。因此，如果在 `azure-functions` 插件属性中没有找到相关授权凭证，此时元数据中的授权凭证就会发挥作用。
+
+:::note
+
+授权方式优先级排序如下：
+
+- 首先，`azure-functions` 插件在 Apache APISIX 代理的请求头中寻找 `x-functions-key` 或 `x-functions-clientid` 键。
+- 如果没有找到，`azure-functions` 插件会检查插件属性中的授权凭证。如果授权凭证存在，`azure-functions` 插件会将相应的授权标头添加到发送到 Azure cloud function 的请求中。
+- 如果在 `azure-functions` 插件属性中没有找到授权凭证，Apache APISIX 将获取插件元数据配置并使用 API 密钥。
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/azure-functions.md:64]:
```suggestion
1. 首先，`azure-functions` 插件在 APISIX 代理的请求头中寻找 `x-functions-key` 或 `x-functions-clientid` 键。
2. 如果没有找到，`azure-functions` 插件会检查插件属性中的授权凭证。如果授权凭证存在，`azure-functions` 插件会将相应的授权标头添加到发送到 Azure Functions 的请求中。
3. 如果未配置 `azure-functions` 插件的授权凭证属性，APISIX 将获取插件元数据配置并使用 API 密钥。
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 11]
```diff
@@ -0,0 +1,202 @@
...
+content-type: text/plain; charset=utf-8
+...
+Hello, APISIX
+```
+
+### 配置路径转发
+
+`azure-functions` 插件在代理请求到 Azure Functions 上游时也支持 URL 路径转发。基本请求路径的扩展被附加到插件配置中指定的 `function_uri` 字段上。
+
+:::info 注意
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/azure-functions.md:146]:
```suggestion
:::info IMPORTANT
```

File: docs/zh/latest/plugins/azure-functions.md [hunk 12]
```diff
@@ -0,0 +1,202 @@
...
+            }
+        }
+    },
+    "uri": "/azure"
+}'
+```
+
+通过上述示例配置插件后，任何对 `/azure` URI 的请求（`HTTP/1.1`、`HTTPS`、`HTTP2`）都将调用已配置的 Azure Functions 的 URI，并且会将响应信息返回给客户端。
+
+下述命令的含义是：Azure Function 从请求中获取 `name` 参数，并返回一条 `"Hello $name"` 消息：
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/azure-functions.md:98]:
```suggestion
下述命令的含义是：Azure Functions 从请求中获取 `name` 参数，并返回一条 `"Hello $name"` 消息：
```

Keep

File: docs/zh/latest/plugins/azure-functions.md [hunk 13]
```diff
@@ -0,0 +1,202 @@
...
+            "authorization": {
+                "apikey": "<Generated API key to access the Azure-Function>"
+            }
+        }
+    },
+    "uri": "/azure/*"
+}'
+```
+
+通过上述示例配置插件后，任何访问 `azure/HttpTrigger1` 的请求都会调用 Azure Function 并转发附加的参数。
```
Comment (lunarwhite) [inline docs/zh/latest/plugins/azure-functions.md:169]:
```suggestion
通过上述示例配置插件后，任何访问 `azure/HttpTrigger1` 的请求都会调用 Azure Functions 并转发附加的参数。
```

**Review by lunarwhite — COMMENTED** [review]
Other LGTM!

**Review by SylviaBABY — DISMISSED** [review]

**Review by juzhiyuan — DISMISSED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**